frontend/src/utils/api-client.js

```diff
@@ -44,7 +44,7 @@ const search = question => client("question/", {body: {question}})
             return a;
         })
         answers.sort((a1, a2) => a2.prob - a1.prob);  // They're compared rounded, but it doesn't matter.
-        return answers.filter((a, i) => (i < 1) || (a.prob >= 10));
+        return answers.filter((a, i) => (i < 3) || (a.prob >= 10));
```

Comment:
Do we always wanna show 3 answers? Maybe we should default to 0? Thoughts?

Reply:
Yes, I changed to only show answers with prob higher than 10

Reply:
But this code will always show the first 3 (when the index is 0, 1, or 2). Is that intended?

Reply:
This is the `Outdated` version.